templates/todo/projects/python-mongo/.repo/terraform/infra/keyvault.tf

```diff
@@ -0,0 +1,53 @@
+# ------------------------------------------------------------------------------------------------------
+# DEPLOY AZURE KEYVAULT
+# ------------------------------------------------------------------------------------------------------
+resource "azurecaf_name" "kv_name" {
+  name          = random_string.resource_token.result
+  resource_type = "azurerm_key_vault"
+  random_length = 0
+  clean_input   = true
+}
+
+resource "azurerm_key_vault" "kv" {
+  name                     = azurecaf_name.kv_name.result
+  location                 = azurerm_resource_group.rg.location

```

Comment:
Where, in these tf files, does the RG get created?

Reply:
in the main.tf https://github.com/Azure/azure-dev/blob/17dc803b0b99c5464f402986f49077e241c8ab64/templates/todo/projects/python-mongo/.repo/terraform/infra/main.tf#L14